usage/cli/src/main/java/brooklyn/cli/ItemLister.java

```diff
@@ -66,6 +68,9 @@
 public class ItemLister {
```

Comment:
Ah, yes I did know about it - it's for us generating documentation (config/attributes/effectors) about all our entities, policies, etc.

Anyway, the comment about docs doesn't need addressed in this PR.